tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -1207,14 +1259,14 @@ public function test_level3_data_bundle() {
 
 	public function test_level3_data_bundle_for_orders_with_more_than_200_items() {
 		$this->mock_wcpay_account->method( 'get_account_country' )->willReturn( 'US' );
-		$mock_order   = $this->mock_level_3_order( '98012', true, 1, 500 );
+		$mock_order   = $this->mock_level_3_order( '98012', true, true, 1, 500 );
```

Comment:
Can we minimize changes here: `$mock_order = $this->mock_level_3_order( '98012', true, false, 1, 500 );`?

Reply:
Addressed in [5c3d486](https://github.com/Automattic/woocommerce-payments/pull/4244/commits/5c3d4866fba7668abdf766121e212dbc94eb87f7)